src/StringLiteralGenerator/StringLiteralGenerator.csproj

```diff
@@ -6,6 +6,7 @@
     <Nullable>enable</Nullable>
     <IsRoslynComponent>true</IsRoslynComponent>
     <IncludeBuildOutput>false</IncludeBuildOutput>
+		<DevelopmentDependency>true</DevelopmentDependency>
```

Comment:
tab?

(VS 2022 uses the same style as XML for csproj and its indent style is tab. Please format manually to replace tab to 2 spaces, or add .editorconfig `[*.csproj] indent_style = space`.)

Reply:
Sorry for the indentation.
I fix it.